components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -383,6 +384,10 @@ public class OAuth2Util {
 
     private static final String BASIC_AUTHORIZATION_PREFIX = "Basic ";
 
+    private static final List<String> SAAS_APP_IDS = Arrays.asList(
```

Comment:
```suggestion
    private static final List<String> PORTAL_APP_IDS = Arrays.asList(
```
Console & MyAccount are not SaaS apps. In some places those apps are identified as portal apps. Hence we can have that or any convenient name.

Reply:
Updated with [5fc4333](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2458/commits/5fc43339d35766c8996f2ff612eeda9ea75f37e7)